src/data/list/forall2.lean

```diff
@@ -55,10 +55,10 @@ begin
   { assume h, subst h, exact forall₂_refl _ }
 end
 
-@[simp] lemma forall₂_nil_left_iff {l} : forall₂ r nil l ↔ l = nil :=
+@[simp, priority 900] lemma forall₂_nil_left_iff {l} : forall₂ r nil l ↔ l = nil :=
```

Comment:
Why is this in this PR?

Reply:
Because they overlap with another simp lemma:
https://github.com/leanprover-community/mathlib/blob/67f3fdeaffa7334cadce4e3689899e355d65d079/src/data/list/defs.lean#L279

The `forall₂.nil` lemma is a constructor, and therefore the linter did not catch this issue before.

Reply:
OK, thanks.